fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -42,7 +42,7 @@
 @WebListener("IBM FHIR Server Servlet Context Listener")
 public class FHIRServletContextListener implements ServletContextListener {
```

Comment:
Is this only a formatting change?  I am not seeing where the code changed.

Reply:
yeah, I made some code changes here and then don't think it's necessary, so just minor adjust to the format.